src/routes/(index)/docs/config/nvchad_ui.mdx

```diff
@@ -47,7 +47,7 @@ M.ui = {
 ```
 <br/>
 
-It is recommended to check the list of modules in [our `statusline` modules file](https://github.com/NvChad/ui/blob/main/lua/nvchad_ui/statusline/modules.lua). In the above code, you can see that we want to print "bruh" next to the mode module, in the statusline. In order to add highlight group to your text, do:
+It is recommended to check the list of modules in [our `statusline` modules file](https://github.com/NvChad/ui/blob/v2.0/lua/nvchad_ui/statusline). In the above code, you can see that we want to print "bruh" next to the mode module, in the statusline. In order to add highlight group to your text, do:
```

Comment:
Another suggestion, depending on the branching strategy you plan to use, would be to use `https://github.com/NvChad/ui/blob/HEAD/lua/nvchad_ui/statusline`. This way it'll always link to the default branch, and if v3.0 becomes the main branch, the docs will update to that automatically, and will only break if the file structure breaks, as was the case here between v1 and v2